pkg/controller/pagerdutyintegration/clusterdeployment_created.go

```diff
@@ -129,6 +129,21 @@ func (r *ReconcilePagerDutyIntegration) handleCreate(pdclient pd.Client, pdi *pa
 			return err
 		}
 	}
+	// In the event of a ClusterRelocation, owner-references will be lost. Check and re-instate.
+	cm := &corev1.ConfigMap{}
+	err = r.client.Get(context.TODO(), types.NamespacedName{Name: configMapName, Namespace: cd.Namespace}, cm)
+	if err != nil {
+		return err
+	}
+	if len(cm.OwnerReferences) == 0 {
+		if err = controllerutil.SetControllerReference(cd, cm, r.scheme); err != nil {
+			r.reqLogger.Error(err, "Error setting controller reference on configmap")
+			return err
+		}
+		if err := r.client.Update(context.TODO(), cm); err != nil {
```

Comment:
Update the ownerReferences or recreate the full resource, which one is the better approach do you think?

Reply:
My main reason for `Update()` was again to avoid a situation where the incoming operator would need to delete the SyncSet (which would remove the managed secret on the remote cluster). 

I personally prefer the approach of deleting everything and letting it be recreated from scratch, but.. I also was hoping to avoid any changes at all to remote cluster during the migration process, so that drove me down this path.